audio-worklet/src/bitcrusher-controller.js

```diff
@@ -30,112 +30,112 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+      this.setGain.bind(this), {type: "range", min: 0, max: 1,
+      step: 0.01, default: 0.5, name: "Volume"});
 
-		const songUrl = "audio/it-from-bit.mp3";
-		let that = this;
+    const songUrl = "audio/it-from-bit.mp3";
+    let that = this;
 
-		// Fetch song from server and decode into buffer
-		fetch(songUrl).then((response) => { 
-			return response.arrayBuffer();})
-				.then((song) => { 
-					that.context.decodeAudioData(song)
-						.then((buffer) => {
-		  				that.songBuffer = buffer;
-		  				that.sourceButton_.enable();
-		  			}, (message) => {console.error(message)});
-		});
-	}
+    // Fetch song from server and decode into buffer
+    fetch(songUrl).then((response) => { 
+      return response.arrayBuffer();})
+        .then((song) => { 
+          that.context.decodeAudioData(song)
+            .then((buffer) => {
+              that.songBuffer = buffer;
+              that.sourceButton_.enable();
+            }, (message) => {console.error(message)});
+    });
+  }
 
-	/**
-	 * Change bit depth (bound to event listener by a ParamController)
-	 * @param {Number} value the new bit depth
-	 */
-	setBitDepth(value) {
-		this.bitDepth_ = value || this.bitDepth_;
-		this.bitcrusher_.bitDepth = this.bitDepth_;
-	}
+  /**
+   * Change bit depth (bound to event listener by a ParamController)
+   * @param {Number} value the new bit depth
+   */
+  setBitDepth(value) {
+    this.bitDepth_ = value || this.bitDepth_;
+    this.bitcrusher_.bitDepth = this.bitDepth_;
+  }
  
   /**
-	 * Change sample rate reduction (bound to event listener by a ParamController)
-	 * @param {Number} value the new sample rate reduction
-	 */
-	setReduction(value) {
-		this.reduction_ = value || this.reduction;
-		this.bitcrusher_.reduction = this.reduction_;
-	}
+   * Change sample rate reduction (bound to event listener by a ParamController)
+   * @param {Number} value the new sample rate reduction
+   */
+  setReduction(value) {
+    this.reduction_ = value || this.reduction;
+    this.bitcrusher_.reduction = this.reduction_;
+  }
  
   /**
- 	 * Change the volume
-	 * @param {Number} value the new gain
-	 */
-	setGain(value) {
-		this.gainNode_.gain.value = value || 0.5;
-	}
+   * Change the volume
+   * @param {Number} value the new gain
+   */
+  setGain(value) {
+    this.gainNode_.gain.value = value || 0.5;
+  }
 
-	/**
-	 * Configure and initiate audio processing
-	 */
-	start() {
-		// Play loaded song, running samples through a bitcrusher under user control
-		this.song_ = this.context.createBufferSource(); 
-		this.song_.buffer = this.songBuffer; 
-		this.song_.start();
-		this.bitDepthParam_.enable();
-		this.reductionParam_.enable();
+  /**
+   * Configure and initiate audio processing
+   */
+  start() {
+    // Play loaded song, running samples through a bitcrusher under user control
+    this.song_ = this.context.createBufferSource(); 
+    this.song_.buffer = this.songBuffer; 
+    this.song_.start();
+    this.bitDepthParam_.enable();
```

Comment:
`bitDepthParam` is really referring to the slider element?  If so, `bitDepthSlider` seems more informative.